### PR TITLE
PLANET-4054: Gutenberg Compatibility: Separate Bootstrap, layout styles and block styles in master theme

### PR DESCRIPTION
**Pull request by pablocubico**
(WIP)

Ref: https://jira.greenpeace.org/browse/PLANET-4054

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,6 @@
 [submodule "react-blocks/src/styles/styleguide"]
 	path = react-blocks/src/styles/styleguide
 	url = https://github.com/greenpeace/planet4-styleguide.git
+[submodule "assets/src/styles/styleguide"]
+	path = assets/src/styles/styleguide
+	url = https://github.com/greenpeace/planet4-styleguide
```
Comment (comzeradd) [inline .gitmodules]: I guess we can remove the old submodule description here.

**Comment by comzeradd** [comment]
Can you also fix [stylelint path](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/develop/.circleci/config.yml#L50)?

**Comment by comzeradd** [comment]
Tested it locally and it seems to work fine. Do you also want to squash some or all the commits before merge?

**Comment by comzeradd** [comment]
Also if `style.css` on the root folder is not used, we can delete it.

**Comment by comzeradd** [comment]
This looks good, but needs rebase. Not sure if you want to remove the lodash/babel fix after all, since this wasn't the root cause of the bug.

**Comment by pablocubico** [comment]
@comzeradd rebased, lodash commit removed!

**Pull request merged**